src/code_0805436C.c

```diff
@@ -182,17 +206,17 @@ u32 CreateRandomItemDrop(Entity* arg0, u32 arg1) {
          ITEM_ENEMY_BEETLE, ITEM_NONE,    ITEM_NONE,   ITEM_NONE,           ITEM_NONE,          ITEM_NONE,
      }*/;
 
-    int r0, r1, r2, r4, r5;
+    int r0, r1, rand, summOdds, item;
     u32 r3;
     const Droptable *ptr2, *ptr3, *ptr4;
-    Droptable s0;
+    Droptable droptable;
     r3 = arg1;
     if (gRoomVars.field_0x2 != 1) {
-        ptr2 = &gUnk_08001A1C[0];
-        ptr4 = 0;
+        ptr2 = &gDroptableModifierNone;
+        ptr4 = NULL;
         switch (r3) {
             case 1 ... 12:
-                ptr4 = &gUnk_0800137C[r3];
+                ptr4 = &gDroptablesEnemies[r3];
```

Comment:
I think gEnemyDroptables might be a better naming scheme to avoid avoid confusion 

Same goes for the rest